library/Faktory/JobOptions.hs

```diff
@@ -58,6 +60,7 @@ instance FromJSON JobOptions where
       <*> o .:? "retry"
       <*> o .:? "queue"
       <*> (fmap (Last . Right) <$> o .:? "at")
+      <*> o .:? "custom"
```

Comment:
If I were an API, I would simply have descriptive names for every field :trollface: